modules/custom/dkan_json_form/js/app/README.md

```diff
@@ -0,0 +1,85 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Development
+
+Assumes installation of [DKAN Tools](https://github.com/getkdkan/dkan-tools).
+
+To work locally in development mode
+- Copy your localhost url to `const baseUrl = "";` in the App.js file.
+- `npm start`
+
+When ready to commit your changes
+- Remove the value from `const baseUrl = "";`
+- `rm -R ../build`
+- `npm run build`
+- `dktl drush dkan-json-form:sync`
+- `dktl drush cr`
+- Review the form at `/admin/dkan/dataset`
+
+
+## Available Scripts
```

Comment:
Should we get rid of the documentation created by react app?

Reply:
yes